### PR TITLE
Inventory

### PR DESCRIPTION
**Pull request by matnguyen**
Things that I didn't add (dunno if I should since we are toning down features):

* Drag and drop for equipping items (currently sequential)
* Item icons (currently just changing colors as placeholder)

### CODE REVIEW

File: public/scripts/main.js [hunk 1]
```diff
@@ -244,9 +246,12 @@ const tooltipBox = new Konva.Rect({
   cornerRadius: 10,
 });
 
+const playerInventory = new Inventory();
```
Comment (Fluffisaurus) [inline public/scripts/main.js]: I'm thinking if an instance of `inventory` should be created in `Game` along with the player given that it should be persistent? Thoughts? @matnguyen @Bruno-DaSilva @abhatthal @Mibrer

File: public/scripts/main.js [hunk 2]
```diff
@@ -244,9 +246,12 @@ const tooltipBox = new Konva.Rect({
...
+
 let readyToInteract = false;
 let inFightScene = false;
 let atEndPoint = false;
+let inInventoryWindow = false;
```
Comment (Fluffisaurus) [inline public/scripts/main.js]: These were moved to `MapControl` as well

**Comment by Mibrer** [comment]
Looks good on my end

**Comment by matnguyen** [comment]
@Fluffisaurus I'm unsure how I should abstract the "inventory scene" since it is integrated into the Inventory class

**Comment by Fluffisaurus** [comment]
> @Fluffisaurus I'm unsure how I should abstract the "inventory scene" since it is integrated into the Inventory class

We can do it after we merge everything into develop

**Pull request merged**